src/containers/BuildUnsignedSweepCoin/BuildUnsignedSweepCoin.tsx

```diff
@@ -200,7 +201,7 @@ function Form() {
                     maxPriorityFeePerGas: toWei(maxPriorityFeePerGas),
                   },
                   replayProtectionOptions: {
-                    chain: bitGoEnvironment === 'prod' ? 1 : 5,
+                    chain: coin === 'ethw' ? NaN : bitGoEnvironment === 'prod' ? 1 : 5,
```

Comment:
@fperalta-simtlix why are we using NaN as a value here?

Reply:
added a TODO comment with a Jira task id to update it when ETHw chainid is confirmed.